README.md

```diff
@@ -1,12 +1,48 @@
+* Ubuntu bionic, xenial
+  * arm64, amd64
+* Raspberry Pi OS buster, stretch
+  * armhf
+* Debian buster, stretch
+  * amd64
+* macOS
+  * amd64
+
+Open Horizon **edge cluster** capability helps you manage and deploy workloads from a management hub cluster to remote instances of OpenShift® Container Platform or other Kubernetes-based clusters. Edge clusters are edge nodes that are Kubernetes clusters. An edge cluster enables use cases at the edge, which require colocation of compute with business operations, or that require more scalability, availability, and compute capability than what can be supported by an edge device. Further, it is not uncommon for edge clusters to provide application services that are needed to support services running on edge devices due to their close proximity to edge devices. Open Horizon deploys edge services to an edge cluster, via a Kubernetes operator, enabling the same autonomous deployment mechanisms used with edge devices. The full power of Kubernetes as a container management platform is available for edge services that are deployed by Open Horizon. Currently supported edge cluster architectures: 
+* [OCP on x86_64 platforms](https://docs.openshift.com/container-platform/4.5/welcome/index.html)
+* [K3s - Lightweight Kubernetes](https://rancher.com/docs/k3s/latest/en/)
+* [MicroK8s](https://microk8s.io/docs) on Ubuntu 18.04 (for development and test, not recommended for production)
+
+Currently there is only one example service in this repository that is designed to run on an edge cluster and that is the [nginx-operator](edge/services/nginx-operator).
+
+# Example Services 
+During the management hub installation, several services should have been published into the exchange automatically. The following three command will list the services, patterns, and deployment policies available in your exchange:
+```
+hzn exchange service list IBM/
+hzn exchange pattern list
+hzn exchange deployment listpolicy 
+```
+**Note:** The above commands assume you have `HZN_ORG_ID` set to the created user org (default is `mycluster`), and your `HZN_EXCHANGE_USER_AUTH` set accordingly.
+
+You can find a list of available edge services in this repository located in the [edge/services](edge/services) directory. For the most part, each of the services are boken up into micro-services designed to accomplish one specific task. This makes them easier to incorporate into a wide variety of "top-level" services. 
+
+A good example of a "top-level" service is [cpu2evtstreams](edge/evtstreams/cpu2evtstreams), which has two dependent services ([cpu_percent](edge/services/cpu_percent), and [gps](edge/services/gps)). It uses these two micro-services to gather information about the edge node it is running on and sends it to an instance of IBM Event Streams using `kafkacat`.
+
+Edge examples specific to the Watson IoT Platform are found in [edge/wiotp](edge/wiotp). These examples are not being maintained. 
```

Comment:
This isn't specific to this review, but should we archive or remove the Watson IoT Platform examples at some point?

Reply:
I would certainly like to do something like that. I'd like to get Bruce's input on that as well

Reply:
Yes, i agree.

Reply:
Perfect, I'll take care of this in the very near future